bindings/gumjs/gumv8database.cpp

```diff
@@ -157,16 +157,16 @@ _gum_v8_database_finalize (GumV8Database * self)
 GUMJS_DEFINE_FUNCTION (gumjs_database_open)
 {
   gchar * path;
+  gint flags;
   sqlite3 * handle;
   gint status;
   Local<Object> object;
 
-  if (!_gum_v8_args_parse (args, "s", &path))
+  if (!_gum_v8_args_parse (args, "si", &path, &flags))
```

Comment:
I guess `guint` / `'u'` would be more appropriate?

Reply:
well, i thought about that too... but from the docs it seems `sqlite3_open_v2` consumes signed flags: https://sqlite.org/c3ref/open.html

Reply:
Ahh, nice catch!